transfer-archive/app/services/rclone_service.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import sys
+
+from app.connectors.sas_generator.sas_generator_client import SASGeneratorClient
+from app.services.models import RcloneS3Config, RcloneAzureConfig
+from app.services.sas_service import generate_sas_token
+from app.helpers.filewriter import write_rclone_config
+from app.settings import Settings
+
+logger = logging.getLogger(__name__)
+
+
+def create_rclone_copy_command(s3_remote: RcloneS3Config, azure_remote: RcloneAzureConfig):
+    settings = Settings()
+    s3 = f"{s3_remote.name}:{settings.bucket_name}/{settings.s3_path}"
+    azure = f"{azure_remote.name}:{settings.azure_container}"
+    return ["rclone", "copy", azure, s3, "-P"]
+
+
+async def setup_rclone_remotes(sas_generator_client: SASGeneratorClient):
+    settings = Settings()
+
+    sas_token = await generate_sas_token(settings.azure_container, sas_generator_client)
+    sas_url = f"{settings.azure_base_url}/{settings.azure_container}?{sas_token.sas_token}"
+    azure_rclone_config = RcloneAzureConfig(sas_url=sas_url)
+
+    s3_rclone_config = RcloneS3Config(access_key_id=settings.iam_access_key_id,
+                                      secret_access_key=settings.iam_secret_access_key,
+                                      endpoint=settings.s3_endpoint_url)
```

Comment:
Så får vi med trailing comma
```suggestion
    s3_rclone_config = RcloneS3Config(
        access_key_id=settings.iam_access_key_id,
        secret_access_key=settings.iam_secret_access_key,
        endpoint=settings.s3_endpoint_url,
    )
```